### PR TITLE
upgrade node-fetch to 2.6.1 to fix vulnerability

### PR DESCRIPTION
**Pull request by wpc**
## Motivation
Vulnerability reference(s):
https://github.com/node-fetch/node-fetch/security/advisories/GHSA-w7rc-rwvf-8q5r
https://nvd.nist.gov/vuln/detail/CVE-2020-15168
https://github.com/advisories/GHSA-w7rc-rwvf-8q5r

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan
```
yarn upgrade node-fetch@2.6.1
yarn install
yarn start
```
the website is still working
## Related Issues and PRs
n/a

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -10,10 +10,11 @@
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.62",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.62",
+    "@docusaurus/core": "^2.0.0-alpha.64",
```
Comment (omry) [inline website/package.json:13]: You also upgraded Docusaurus to the latest release (from today).
this seems unintentional.
Did you verify that the website is functional?

**Comment by omry** [comment]
Looking at the website preview, it seems fine.

File: website/package.json (hunk 1 above)
Comment (wpc) [reply website/package.json:13]: Thank you for merging. I found if we upgrade node-fetch to 2.6.1 the docusaurus libs are upgraded to alpha.64 anyway in the yarn.lock, so I put the new version there in packages.json to make it explicit. I did check the website after upgrade, it seems looks fine. But I am not familiar with all the functionalities there. Do you have some specific checks every time you update website?

Comment (omry) [reply website/package.json:13]: I am just making sure that is builds and comes up. Historically this has been enough to catch any serious problem.

**Comment by omry** [comment]
for future reference, this links to a website preview:
image

**Pull request merged**